osu.Game/Graphics/DrawableJoinDate.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+
+namespace osu.Game.Graphics
+{
+    public class DrawableJoinDate : DrawableDate
+    {
+        private readonly DateTimeOffset date;
+
+        public DrawableJoinDate(DateTimeOffset date) : base(date)
+        {
+            this.date = date;
+        }
+
+        protected override string Format() => Text = string.Format($"{date:MMMM yyyy}");
+
+        public override string TooltipText => string.Format($"{date:d MMMM yyyy}");
```

Comment:
Format should be `MMMM d, yyyy` to match web.

![image](https://user-images.githubusercontent.com/6215387/40884594-eaeb3962-66ec-11e8-8b73-aa80e208bd4e.png)

Reply:
This class should probably also handle the "Here since the beginning" text. As it is when hovering over that text in the profile overlay, no date is shown.

Reply:
1: I guess I will have to check if there's some culture-specific formatting, 'cause the text varies language-to-language.

Reply:
It does right now, I'm saying `DrawableJoinDate` should instead.